scripts/svm/simpleFill.ts

```diff
@@ -50,7 +50,7 @@ async function fillV3Relay(): Promise<void> {
   const inputAmount = new BN(resolvedArgv.inputAmount);
   const outputAmount = new BN(resolvedArgv.outputAmount);
   const originChainId = new BN(resolvedArgv.originChainId);
-  const depositId = intToU8Array32(resolvedArgv.depositId);
+  const depositId = intToU8Array32(new BN(resolvedArgv.depositId));
```

Comment:
I don't think BN constructor automatically detects and parses hex strings

Reply:
it's assumed its just a large number in a string so should be fine.